tests/ui/option_env_unwrap.rs

```diff
@@ -9,6 +9,7 @@ use proc_macros::{external, inline_macros};
 fn main() {
     let _ = option_env!("PATH").unwrap();
     let _ = option_env!("PATH").expect("environment variable PATH isn't set");
+    let _ = option_env!("Y").unwrap(); // This test only works if you don't have a __Y__do_not_use env variable in your environment.
```

Comment:
Is the comment outdated and should be

```suggestion
    let _ = option_env!("Y").unwrap(); // This test only works if you don't have a Y env variable in your environment.
```